webextensions/api/windows.json

```
@@ -656,7 +656,8 @@
               "firefox": {
                 "notes": [
                   "'url' and 'tabId options can't both be set together.",
-                  "The returned 'Window' object contains the 'tabs' property only from version 52 onwards."
+                  "The returned 'Window' object contains the 'tabs' property only from version 52 onwards.",
+                  "Firefox currently ignores the 'focused:false' option."
```

Thanks for this PR, @caitmuenster. I trust the other review here on the substance of the note, but I do have a couple of style concerns.

- "Currently" is a bit of a moving target and difficult to maintain (we don't reconfirm every note for every release). Can we constrain this information at all? For example, if this is a new behavior, can it be anchored to a specific version number (e.g., "From version X, Firefox ignores the `focused: false` option."). Or if it's been true from inception, can we drop the word "currently"?
- I think it'd be nice to put the quoted text in code format (even though the other notes didn't do that)

```suggestion
                  "Firefox currently ignores the <code>focused: false</code> option."
```